test/index.tsx

```diff
@@ -2,7 +2,7 @@ import * as assert from 'assert';
 import { h } from 'snabbdom/h';
 import { VNode } from 'snabbdom/vnode';
 import { assertLooksLike, looksLike, Wildcard } from '../src/index';
-import Snabbdom from 'snabbdom-pragma';
+import * as Snabbdom from 'snabbdom-pragma';
```

Comment:
Or we need TS compiler option `esModuleInterOp` is enabled.

ref: https://unional.gitbooks.io/typescript/content/pages/06-files-and-projects/compiler-options.html#esmoduleinterop